etc/k8s/cluster_init.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+set -euo pipefail
+
+function main() {
+    timeout=600
+
+    # Initial setup for all nodes.
+    run_job kernelpatch-node-daemonset.yaml kernelpatch-node $timeout
+    run_job node-init-daemonset.yaml node-init $timeout
+    # Start transitd on all nodes.
+    kubectl apply -f transitd-daemonset.yaml
+    validate "running" transitd-daemonset.yaml transitd $timeout
+    # Load transit xdp program on main interface of all nodes.
+    run_job load-transit-xdp-daemonset.yaml load-transit-xdp $timeout
+}
+
+function run_job() {
+    running="running"
+    done="done"
+
+    kubectl apply -f $1
+    validate $running $1 $2 $3
+    validate $done $1 $2 $3
+    kubectl delete -f $1
```

Comment:
Let's assign $1, $2, $3 to variables for readability

Reply:
Fixed in the latest commit!